deploy.sh

```diff
@@ -51,10 +51,12 @@ docker run --rm -v $PWD:${CODEBASE_PATH} \
 
 # Use google/cloud-sdk image to deploy the cloud function
 # TODO: is there a smaller version of this image we can use?
+# GCP_PROJECT env var was set automatically for nodejs 6 and 8, and the runtime code uses it.
+# but it is not set automatically for nodejs 10; therefore we set it here during deploy.
 docker run --rm -v $PWD:${CODEBASE_PATH} \
     -e BASE_URL="https://us-central1-broad-dsde-${ENVIRONMENT}.cloudfunctions.net" \
-    google/cloud-sdk:220.0.0 /bin/bash -c \
+    google/cloud-sdk:308.0.0 /bin/bash -c \
```

Comment:
220 didn't support `--set-env-vars` in the GA version of `functions deploy`, so I'm upgrading it